src/wp-admin/about.php

```diff
@@ -53,80 +53,76 @@
 			<a href="contribute.php" class="nav-tab"><?php _e( 'Get Involved' ); ?></a>
 		</nav>
 
-		<div class="about__section">
+		<div class="about__section has-1-column">
 			<div class="column">
-				<h2>
-					<?php
-					printf(
-						/* translators: %s: Version number. */
-						__( 'Welcome to WordPress %s' ),
-						$display_version
-					);
-					?>
-				</h2>
-				<p class="is-subheading">
-					<?php _e( 'WordPress 6.7 debuts the modern Twenty Twenty-Five theme, offering ultimate design flexibility for any blog at any scale. Control your site typography like never before with new font management features. The new Zoom Out feature lets you design your site with a macro view, stepping back from the details to bring the big picture to life.' ); ?>
-				</p>
+				<h2><?php _e( 'A release polished to a high sheen.' ); ?></h2>
+				<p class="is-subheading"><?php _e( 'WordPress 6.8 polishes and refines the tools you use every day, making your site faster, more secure, and easier to manage.' ); ?></p>
+				<p><?php _e( 'The Style Book now has a structured layout and works with Classic themes, giving you more control over global styles.' ); ?></p>
+				<p><?php _e( 'Speculative loading speeds up navigation by preloading links before users navigate to them, while bcrypt hashing strengthens password security automatically.' ); ?></p>
+				<p><?php _e( 'Behind the scenes, database optimizations improve performance, and a new security warning system helps prevent vulnerabilities before they become a problem.' ); ?></p>
 			</div>
 		</div>
 
 		<div class="about__section has-2-columns">
 			<div class="column is-vertically-aligned-center">
-				<h3><?php _e( 'Introducing Twenty Twenty-Five' ); ?></h3>
+				<h3><?php _e( 'The Style Book gets a cleaner look—and a few new tricks' ); ?></h3>
 				<p>
-					<strong><?php _e( 'Endless possibility without complexity' ); ?></strong><br />
-					<?php _e( 'Twenty Twenty-Five offers a flexible, design-focused theme that lets you build stunning sites with ease. Tailor your aesthetic with an array of style options, block patterns, and color palettes. Pared down to the essentials, this is a theme that can truly grow with you.' ); ?>
+					<?php _e( 'The Style Book has a new, structured layout and clearer labels, to make it even easier to edit colors, typography—almost all your site styles—in one place.' ); ?>
+				</p>
+				<p>
+					<?php
+					if ( true ) {
+						printf(
+							/* translators: %s is a direct link to the Style Book. */
+							__( 'Plus, now you can see it in Classic themes that have editor-styles or a theme.json file. Find the Style Book under <a href="%s">Appearance > Design</a> and use it to preview your theme&#8217;s evolution, as you edit CSS or make changes in the Customizer.' ),
+							'#'
+						);
```

Comment:
I'll come back to update this link, the text will also need to change based on block theme or not, since block themes don't have "Appearance > Design". Or maybe this part should be hide for block theme users?